assets/collage.css

```diff
@@ -108,26 +108,27 @@
 
 .collage-card {
   background: rgb(var(--color-background));
-  border-radius: var(--media-radius);
-  border: var(--media-border-width) solid rgba(var(--color-foreground), var(--media-border-opacity));
+  border-radius: var(--card-corner-radius);
+  border: var(--card-border-width) solid rgba(var(--color-foreground), var(--card-border-opacity));
   height: 100%;
+  padding: var(--card-image-padding);
   position: relative;
 }
 
 .collage-card:after {
-  border-radius: var(--media-radius);
-  box-shadow: var(--media-shadow-horizontal-offset) var(--media-shadow-vertical-offset) var(--media-shadow-blur-radius) rgba(var(--color-shadow), var(--media-shadow-opacity));
+  border-radius: var(--card-corner-radius);
+  box-shadow: var(--card-shadow-horizontal-offset) var(--card-shadow-vertical-offset) var(--card-shadow-blur-radius) rgba(var(--color-shadow), var(--card-shadow-opacity));
   content: '';
   position: absolute;
-  width: calc(var(--media-border-width) * 2 + 100%);
-  height: calc(var(--media-border-width) * 2 + 100%);
-  top: calc(var(--media-border-width) * -1);
-  left: calc(var(--media-border-width) * -1);
+  width: calc(var(--card-border-width) * 2 + 100%);
+  height: calc(var(--card-border-width) * 2 + 100%);
+  top: calc(var(--card-border-width) * -1);
+  left: calc(var(--card-border-width) * -1);
   z-index: -1;
 }
 
 .collage-card .media {
-  border-radius: calc(var(--media-radius) - var(--media-border-width));
+  border-radius: calc(var(--card-corner-radius) - var(--card-border-width));
```

Comment:
Good catch! 🙇‍♀️

Reply:
I addressed the focus issue you mention. Curious to hear what you think of it as you have context on it I think or have dealt with it.

I just need to have the same styling for focus when it's no-js 🤔  but ill look into it next week 🙂

Reply:
So actually the focus-within pattern on cards is going to change with Kai's update [here](https://github.com/Shopify/dawn/pull/1094) He's utilizing a pseudo element of the `<a>` to create the outline so we don't need focus-within. It's potentially something we can handle for video later, but it will eventually need to be updated.

Reply:
Create a small PR to update it: https://github.com/Shopify/dawn/pull/1148